demo/emails/yelp-recent-login.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <EHtml>
+    <EHead />
+    <EPreview>Yelp recent login</EPreview>
+    <EBody :style="main">
+      <EContainer>
+        <ESection :style="logo">
+          <EImg src="/yelp-logo.png" />
+        </ESection>
+
+        <ESection :style="content">
+          <EImg width="620" src="/yelp-header.png" />
+
+          <ERow
+            :style="{
+              ...boxInfos,
+              paddingBottom: 0,
+            }"
+          >
+            <EColumn>
+              <EHeading
+                style="font-size: 32; font-weight: bold; text-align: center"
+              >
+                Hi {{ userFirstName }},
+              </EHeading>
+              <EHeading
+                as="h2"
+                :style="{
+                  fontSize: '26px',
+                  fontWeight: 'bold',
+                  textAlign: 'center',
+                }"
+              >
+                We noticed a recent login to your Yelp account.
+              </EHeading>
+
+              <EText :style="paragraph">
+                <b>Time: </b>
+                {{ formattedDate }}
+              </EText>
+              <EText :style="{ ...paragraph, marginTop: '-5px' }">
+                <b>Device: </b>
+                {{ loginDevice }}
+              </EText>
+              <EText :style="{ ...paragraph, marginTop: '-5px' }">
+                <b>Location: </b>
+                {{ loginLocation }}
+              </EText>
+              <EText
+                :style="{
+                  color: 'rgb(0,0,0, 0.5)',
+                  fontSize: 14,
+                  marginTop: '-5px',
+                }"
+              >
+                *Approximate geographic location based on IP address: {loginIp}
+              </EText>
+
+              <EText :style="paragraph">
+                If this was you, there's nothing else you need to do.
+              </EText>
+              <EText :style="{ ...paragraph, marginTop: '-5px' }">
+                If this wasn't you or if you have additional questions, please
+                see our support page.
+              </EText>
+            </EColumn>
+          </ERow>
+          <ERow :style="{ ...boxInfos, paddingTop: '0' }">
+            <EColumn :style="containerButton" col-span="{2}">
+              <EButton href="#" :style="button">Learn More</EButton>
+            </EColumn>
+          </ERow>
+        </ESection>
+
+        <ESection :style="containerImageFooter">
+          <EImg width="620" src="/yelp-footer.png" />
+        </ESection>
+
+        <EText
+          :style="{
+            textAlign: 'center',
+            fontSize: 12,
+            color: 'rgb(0,0,0, 0.7)',
+          }"
+        >
+          © 2022 | Yelp Inc., 350 Mission Street, San Francisco, CA 94105,
+          U.S.A. | www.yelp.com
+        </EText>
+      </EContainer>
+    </EBody>
+  </EHtml>
+</template>
+
+<script setup lang="ts">
+const props = defineProps({
```

Comment:
Here we can define the props using typescript features? Just to give more examples using ts.